tests/InputFiles/ScalarWave/SphericalWave3D.yaml

```diff
@@ -0,0 +1,54 @@
+# Distributed under the MIT License.
```

Comment:
The generalization everywhere looks good. To reduce the number of executables we build, for the other evolution systems we only have 1 input file for each dimension. Would you mind removing this input file so that we don't build the extra executable in CI?

Reply:
Makes sense, removed this input file. 

Going forward, we might have even more executables that can be compiled but we dont necessarily want to build and test in CI. The simplest example would be of closely-related problems, for which testing a select number is sufficient for detecting problems, such as the case here. For these, should we think of storing input files outside of `tests` somewhere? Maybe in `support` or even in `docs` as extra examples, as they'll still be useful for tutorials etc?

Reply:
Yep, the plan is to have a set of input files that only get run on develop maybe once a week. Or to have a separate job for just testing those input files so as to keep total build time down